src/Game.cxx

```diff
@@ -96,6 +96,37 @@ bool Game::initialize()
   return true;
 }
 
+void Game::newUI()
+{
+  m_UILoopMQ.push(ActivitySwitchEvent{ActivityType::MainMenu});
+  m_GameLoopMQ.push(ActivitySwitchEvent{ActivityType::MainMenu});
+  SDL_Event event;
+  for(;;)
+  {
```

Comment:
This will need to be implemented gradually. Ideally, it will just forward SDL events to our GameLoop.

Reply:
Pt II